.reuse/dep5

```diff
@@ -0,0 +1,64 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
This file seems fine to me, but I'm concerned about it falling out of date. Was this file generated by some mechanism, and if so, is there a way we can wire it into our GitHub Actions to ensure this file remains up-to-date?

Reply:
The file is handwritten starting from a template from a hacky script. It's not uncommon for such files to be a bit out of date and sometimes authors don't bother with putting their full name and email address, but this does not limit or voids author's rights: copyright law applies automatically and the file is a reminder for end users, a way to help legal teams check authorship, and a way to recognize people's efforts.

Reply:
FWIW `reuse lint` can be run automatically in CI and authors can be suggested to update the file when contributing. Some projects use `AUTHORS.md` or `CONTRIBUTORS.md` similarly.